packages/rich-text/src/join.js

```diff
@@ -11,7 +11,7 @@ import { normaliseFormats } from './normalise-formats';
  * string. This is similar to `Array.prototype.join`.
  *
  * @param {Array}         values    An array of values to join.
- * @param {string|Object} separator Separator string or value.
+ * @param {?(string|Object)} separator Separator string or value.
```

Comment:
Hm, it looks like `separator` is required here?

Reply:
The `separator` arg is given a default value of `''` if none is provided, so I guess is optional? For reference: [current docs](https://github.com/WordPress/gutenberg/tree/master/packages/rich-text#join) also mark it as that can be `null`.

Reply:
Ah, correct.